src/gatling/simulations/uk/gov/hmcts/ccd/corecasedata/scenarios/manageCaseAssignments/casesharing.scala

```diff
@@ -0,0 +1,259 @@
+  //   .check(jsonPath("$.id").saveAs("caseId"))
+  // )
+
+  //  .check(status.saveAs("statusvalue")))
+  //  .doIf(session=>session("statusvalue").as[String].contains("201")) {
+  //    exec {
+  //      session =>
+  //        val fw = new BufferedWriter(new FileWriter("EmailandCaseID.csv", true))
+  //        try {
+  //          fw.write(session("userEmail").as[String] + ","+session("caseId").as[String] + "\r\n")
+  //        }
+  //        finally fw.close()
+  //        session
+  //    }
+  //  }
+
+  //.pause(Environment.constantthinkTime)
+
+  val CaseShareLoginLarge =
+
+    exec(http("02_ACA_GetS2SToken")
+      .post(Environment.s2sUrl + "/testing-support/lease")
+      .header("Content-Type", "application/json")
+      .body(StringBody("{\"microservice\":\"xui_webapp\"}"))
+      .check(bodyString.saveAs("bearerTokenShare")))
+
+      //.feed(feedCSUserDataLarge)
+
+      .exec(http("02_ACA_OIDC01_Authenticate")
+        .post(Environment.idamAPI + "/authenticate")
+        .header("Content-Type", "application/x-www-form-urlencoded")
+        .formParam("username", "${email}")
+        .formParam("password", "${password}")
+        .formParam("redirectUri", Environment.xuiMCUrl)
+        .formParam("originIp", "0:0:0:0:0:0:0:1")
+        .check(status is 200)
+        .check(headerRegex("Set-Cookie", "Idam.Session=(.*)").saveAs("authCookie")))
+
+      .exec(http("02_ACA_OIDC02_Authorize_XUI")
+        .post(Environment.idamAPI + "/o/authorize?response_type=code&client_id=" + xuiClientId + "&redirect_uri=" + Environment.xuiMCUrl + "&scope=" + xuiScope).disableFollowRedirect
+        .header("Content-Type", "application/x-www-form-urlencoded")
+        .header("Cookie", "Idam.Session=${authCookie}")
+        .header("Content-Length", "0")
+        .check(status is 302)
+        .check(headerRegex("Location", "code=(.*)&client_id").saveAs("code")))
+
+      .exec(http("02_ACA_OIDC03_Token_XUI")
+        .post(Environment.idamAPI + "/o/token?grant_type=authorization_code&code=${code}&client_id=" + xuiClientId + "&redirect_uri=" + Environment.xuiMCUrl + "&client_secret=" + xuiwebappClientSecret)
+        .header("Content-Type", "application/x-www-form-urlencoded")
+        .check(status is 200)
+        .check(jsonPath("$.access_token").saveAs("access_tokenShare")))
+
+  //  .exec {
+  //      session =>
+  //        println(session("bearerToken").as[String])
+  //        println(session("access_token").as[String])
+  //        session
+  //    }
+
+  val CaseShareLoginSmall =
+
+    exec(http("02_ACA_GetS2SToken")
+      .post("http://rpe-service-auth-provider-perftest.service.core-compute-perftest.internal/testing-support/lease")
```

Comment:
Remove hardcoded URL

Reply:
this code is existing in the repo and not mine - the casesharing.scala file has only been moved into a new folder.

Reply:
I have now removed this file from the repo as the code is being rewritten as part of the manage case PI28